pkg/db/migrations/002_testcase.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TYPE test_result as ENUM ('passed', 'failed', 'skipped', 'pending');
+
+CREATE TABLE IF NOT EXISTS testcases (
```

Comment:
Open question: Is there value in storing which JUnit xml file the row was parsed from? We're quickly going to be parsing multiple files to generate db data.

Reply:
I don't see an issue with storing it. Do you mean the URL, or just the file name?

Reply:
Just the filename -- we can compose a URL with the rest of the data stored within the job.

Reply:
We can bump this to be an incremental change later.